src/core/backend.h

```diff
@@ -121,6 +121,9 @@ class InferenceBackend {
 
   // Map from output name to the model configuration for that output.
   std::unordered_map<std::string, ModelOutput> output_map_;
+
+  // Platform name
+  std::string platform_name_;
```

Comment:
platform name is availabe in 'config_' so don't need to have it here explicitly.

Reply:
Removed